spec/email_spec/helpers_spec.rb

```diff
@@ -14,7 +14,7 @@
       expect do
         expect(self).to(receive(:visit).with('/hello'))
         visit_in_email("Hello!")
-      end.not_to raise_error(EmailSpec::CouldNotFindEmailError)
+      end.not_to raise_error
```

Comment:
@etagwerker I thought it made sense because if any exception other than `EmailSpec::CouldNotFindEmailError` is raised, it gets silenced, as it's only expecting that one exception. This way, all exceptions are caught.

Also, RSpec displays a warning for `expect { }.not_to raise_error` explaining that, so this fixes that as well.